measurement/util.py

```diff
@@ -97,10 +98,10 @@ def parse_insight_opts():
     parser.add_argument("--perf-time", type=int, action="store", default=None,
                         help="Time period of perf recording, in seconds.")
     parser.add_argument("--perf-archive", action="store_true", default=False,
-                        help="Run `perf archive` after collecting data, useful when reading data on another machine. Disabled by default.")
+                        help="Run `perf archive` after collecting data, useful when reading data on another machine, may cause large output file size. Disabled by default.")
 
     parser.add_argument("-l", "--log", action="store_true", default=False,
-                        help="Collect log files in output. PD/TiDB/TiKV logs are included by default.")
+                        help="Collect log files in output. PD/TiDB/TiKV logs are included by default if no other argument given.")
```

Comment:
->  help="Collect log files in output. PD/TiDB/TiKV logs are included by default if no other argument **is** given.")